languages/js/src/Polar.test.ts

```diff
@@ -750,7 +750,7 @@ Type error: can only use \`in\` on an iterable value, this is Number(Integer(2))
     undefined.foo
   in query at line 1, column 1
     undefined.foo
-Application error: Cannot read properties of undefined (reading 'foo') at line 1, column 1`
```

Comment:
What Node version are you on? We run tests on 12.20.0 in CI

Reply:
16.9.1 ??? I changed it to the longest common prefix of the two messages ...